redirects-list.json

```diff
@@ -6,12 +6,13 @@
   "^/((?:deb|rpm)/.+)                       https://s3-us-east-2.amazonaws.com/dvc-s3-repo/$1 303",
   "^/(?:help|chat)/?$                       https://discordapp.com/invite/dvwXA2N 303",
   "^/(?:docs|documentation)(/.*)?$          /doc$1",
-  "^/doc/commands-reference(/.*)?$          /doc/command-reference$1",
+  "^/doc/?$                                 /doc/tutorials/get-started 307",
```

Comment:
should it be home now?

Reply:
Yes, I also noticed this earlier, it's fixed. Thanks